docs/root/version_history/current.rst

```diff
@@ -1,5 +1,5 @@
-1.21.3 (June 9, 2022)
-=====================
+1.21.4 (June 14, 2022)
```

Comment:
nit: should we have some release note to say why we are cutting this?

/wait

Reply:
ill include info about the docker image update...